roles/kafka_broker/tasks/main.yml

```diff
@@ -351,7 +351,17 @@
     owner: root
     group: root
   notify: restart kafka
-  diff: "{{ not mask_sensitive_diff|bool }}"
+  diff: {{ not mask_sensitive_diff|bool }}
+
+- name: Create sysctl directory on Debian distributions
```

Comment:
Is this related to ansible-sanity failure?

Reply:
It was a warning during galaxy_import command. I have rolled back these changes now.

Reply:
I meant the new task `Create sysctl directory on Debian distributions`